src/silicone/multiple_infillers/infill_all_required_emissions_for_openscm.py

```diff
@@ -69,6 +71,9 @@ def infill_all_required_variables(
         reasons for failing this include requesting results at times outside our input
         time range, as well as code bugs.
 
+    cruncher_kwargs : dict
```

Comment:
is this a dict i.e. would one call `infill_all_required_variables({"cruncher_kwarg_1": val_1, "cruncher_kwarg_2": val_2})` or can you just pass kwargs i.e one would call `infill_all_required_variables(cruncher_kwarg_1=val_1, cruncher_kwarg_2=val_2)`

Reply:
Looking through the tests, I think I see how this behaves. To fit with more standard Python, I would update to 

```python
def infill_all_required_variables(
    to_fill,
    database,
    variable_leaders,
    required_variables_list=None,
    cruncher=QuantileRollingWindows,
    output_timesteps=None,
    infilled_data_prefix=None,
    to_fill_old_prefix=None,
    check_data_returned=False,
    **kwargs,
):
    """
    This is function designed to infill all required data given a minimal amount of
    input.

    Parameters
    ----------

    ...

    **kwargs
        Keyword arguments to be passed to the cruncher.
    """
```

Reply:
ok, changed